webroot/img/flags/nap.svg

```diff
@@ -1,4 +1,4 @@
+  <image xlink:href="data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjxzdmcKICAgdmVyc2lvbj0iMS4wIgogICB3aWR0aD0iNjAwIgogICBoZWlnaHQ9IjQwMCIKICAgaWQ9InN2ZzY5OCIKICAgc29kaXBvZGk6ZG9jbmFtZT0iRmxhZ19vZl9OYXBsZXMuc3ZnIgogICB4bWw6c3BhY2U9InByZXNlcnZlIgogICBpbmtzY2FwZTp2ZXJzaW9uPSIxLjIgKGRjMmFlZGFmMDMsIDIwMjItMDUtMTUpIgogICB4bWxuczppbmtzY2FwZT0iaHR0cDovL3d3dy5pbmtzY2FwZS5vcmcvbmFtZXNwYWNlcy9pbmtzY2FwZSIKICAgeG1sbnM6c29kaXBvZGk9Imh0dHA6Ly9zb2RpcG9kaS5zb3VyY2Vmb3JnZS5uZXQvRFREL3NvZGlwb2RpLTAuZHRkIgogICB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiAgIHhtbG5zOnN2Zz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPjxkZWZzCiAgICAgaWQ9ImRlZnM3MDIiIC8+PHNvZGlwb2RpOm5hbWVkdmlldwogICAgIGlkPSJuYW1lZHZpZXc3MDAiCiAgICAgcGFnZWNvbG9yPSIjZmZmZmZmIgogICAgIGJvcmRlcmNvbG9yPSIjMDAwMDAwIgogICAgIGJvcmRlcm9wYWNpdHk9IjAuMjUiCiAgICAgaW5rc2NhcGU6c2hvd3BhZ2VzaGFkb3c9IjIiCiAgICAgaW5rc2NhcGU6cGFnZW9wYWNpdHk9IjAuMCIKICAgICBpbmtzY2FwZTpwYWdlY2hlY2tlcmJvYXJkPSIwIgogICAgIGlua3NjYXBlOmRlc2tjb2xvcj0iI2QxZDFkMSIKICAgICBzaG93Z3JpZD0iZmFsc2UiCiAgICAgaW5rc2NhcGU6em9vbT0iMC4xMzIxODc1IgogICAgIGlua3NjYXBlOmN4PSItMTA5My4xNDQyIgogICAgIGlua3NjYXBlOmN5PSI1NDguNDYzMzYiCiAgICAgaW5rc2NhcGU6d2luZG93LXdpZHRoPSIxOTIwIgogICAgIGlua3NjYXBlOndpbmRvdy1oZWlnaHQ9IjEwMDUiCiAgICAgaW5rc2NhcGU6d2luZG93LXg9Ii05IgogICAgIGlua3NjYXBlOndpbmRvdy15PSItOSIKICAgICBpbmtzY2FwZTp3aW5kb3ctbWF4aW1pemVkPSIxIgogICAgIGlua3NjYXBlOmN1cnJlbnQtbGF5ZXI9InN2ZzY5OCIgLz48cmVjdAogICAgIHdpZHRoPSIzMDAiCiAgICAgaGVpZ2h0PSI0MDAiCiAgICAgZmlsbD0iI0ZGREI3MiIKICAgICBpZD0icmVjdDY5NCIKICAgICBzdHlsZT0iZmlsbDojZmVkYTAwO2ZpbGwtb3BhY2l0eToxIiAvPjxyZWN0CiAgICAgeD0iMzAwIgogICAgIHdpZHRoPSIzMDAiCiAgICAgaGVpZ2h0PSI0MDAiCiAgICAgZmlsbD0iI0YwMTUxNiIKICAgICBpZD0icmVjdDY5NiIKICAgICBzdHlsZT0iZmlsbDojZTUwMDA1O2ZpbGwtb3BhY2l0eToxIiAvPjwvc3ZnPgo=" width="30" height="20"/>
```

Comment:
Replacing the PNG with an SVG was a good idea, but the way you did it was a bit unorthodox.

We use SVG-with-PNG-embedded for many flags so that we can have a uniform SVG-based format even for images where only a PNG is available, and then slowly switch to pure SVG. (Most recently, a few flags were switched to SVG in PR #2865 .)

However, instead of turning SVG-with-PNG-embedded (which had 3154 bytes) into SVG-with-SVG-embedded (with 1958 bytes), you should have used the SVG you got from Wikipedia directly (which has 1310 bytes), only changing the size to 30×20. Changing the size is something you could do with an SVG editor like [Inkscape](https://inkscape.org/)

... in this case I had a look at the Wikipedia SVG and it still contains a lot of unnecessary fluff, so that it could be pared down to 230 bytes (without the newlines I added below for readability)
```svg
<?xml version="1.0"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="30" height="20">
<rect width="15" height="20" fill="#FEDA00"/>
<rect width="15" height="20" fill="#E50005" x="15"/>
</svg>
```
... but this last optimization is not something I'd expect you to do.

Reply:
With the rect commands (for lack of a better term), you’re setting the dimensions, the color, and location of the rectangle, right?

I couldn’t tell what was fluff and what wasn’t, because when I downloaded directly from Wikipedia, it looked like the bottom half was white, but when I ran it through the Data URI converter it gave me only the flag as far as I could tell, without that empty space.

I didn’t realize to change the dimensions before converting it, I’ll do it next time.

Thanks for the pointers and the help, I’ll try it again later today!

Reply:
Since I'm already familiar with Photopea/Photoshop, I used it to scale it down to 30x20. I think that should be do it better!

Reply:
> With the rect commands (for lack of a better term), you’re setting the dimensions, the color, and location of the rectangle, right?

Yes, exactly. For flags that are made entirely out of rectangular shapes, defining the image by hand like this is relatively straightforward.

> I couldn’t tell what was fluff and what wasn’t

Me neither :) Looking at your new version, I noticed that there was another big chunk that I could've removed. (Apparently Photoshop does some file size optimization automatically.)

> Since I'm already familiar with Photopea/Photoshop, I used it to scale it down to 30x20.

That did work, but you still embedded it as an SVG-in-SVG. For images already in SVG format, there should never be a need to use the Data URI converter.

Reply:
Then if I have a SVG file, how would that be turned into Data URI without the converter?

Reply:
Just replace `nap.svg` by the SVG file you created. The Data URI is only required for non-SVG images.